options.go

```diff
@@ -57,6 +67,18 @@ func IgnoreTopFunction(f string) Option {
 	})
 }
 
+// Cleanup sets up a cleanup function that will be executed at the
+// end of the leak check.
+// When passed to [VerifyTestMain], the exit code passed to cleanupFunc
+// will be set to the exit code of TestMain.
+// When passed to [VerifyNone], the exit code will be set to 0.
+// This cannot be passed to [Find].
+func Cleanup(cleanupFunc func(exitCode int)) Option {
+	return optionFunc(func(opts *opts) {
+		opts.teardown = cleanupFunc
```

Comment:
out of curiosity, why are we calling this `teardown` rather than just `cleanup`?

Using the same word in the external API and the internals could be clearer - are we avoiding some other name conflict?

Reply:
that's a good point, I changed everything to `cleanup`.